eventually/authentication/models.py

```diff
@@ -50,6 +66,18 @@ def get_by_id(user_id):
             return None
 
     @staticmethod
+    def get_by_email(user_email):
+        """
+        Returns user by email
+        :param user_email: email by which we need to find the user
+        :return: user object or None if a user with such ID does not exist
+        """
+        try:
+            return CustomUser.objects.get(email=user_email)
+        except CustomUser.DoesNotExist:
+            return None
```

Comment:
maybe return `pass`?

Reply:
Return None is needed by the view functions to properly interpret the result. Pass is not necessary.